.storybook/preview.js

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Please integrate addon-viewport for mobile responsive development

https://storybook.js.org/docs/react/essentials/viewport#use-detailed-set-of-devices